aws/config.go

```diff
@@ -497,13 +486,6 @@ func (c *Config) WithLowerCaseHeaderMaps(t bool) *Config {
 	return c
 }
 
-// WithDisableRestProtocolURICleaning sets a config DisableRestProtocolURICleaning value
-// returning a Config pointer for chaining.
-func (c *Config) WithDisableRestProtocolURICleaning(t bool) *Config {
```

Comment:
We can't remove this (breaking change).

Do please add the same deprecation message as we did on the field itself, though.

Reply:
Gotcha, just over-understand deprecate